docs/docker-integration/cloudformation.rst

```diff
@@ -0,0 +1,167 @@
+.. _cloudformation:
+
+.. raw:: html
+
+    <style>
+        .toctree-wrapper { display:none; }
+    </style>
+
+=========================================================
+Installing Flocker with Swarm on AWS using CloudFormation
+=========================================================
+
+The steps in this guide enable you to quickly deploy a Flocker/Swarm cluster, as illustrated below, which will create four EC2 instances on your AWS account:
+
+.. raw:: html
+
+	<div style="width:80%; margin-left:auto; margin-right:auto; margin-bottom:2em;">
+
+.. figure:: ../images/cloudformation.png
+    :alt: A diagram illustrating a cluster of four AWS EC2 instances running Flocker with Docker Swarm.
```

Comment:
This is a Sphinx thing - the image directive doesn't support `title`. The definition of `alt` suggests that not all browsers are going to display it:

alt : text
Alternate text: a short description of the image, displayed by applications that cannot display images, or spoken by applications for visually impaired users.